STM32G491 missing FDCAN2 peripheral

It appears to be missing in the svd from ST (i.e. it is present in the G484 svd, but not the G491)

Would submit a PR myself but I can't find any examples on how one would add entire peripheral with a patch rather than the individual registers.

Running into this error.

```
[2023-10-26T20:46:34Z ERROR svdtools::cli] by svdtools (0.3.4)
    
    Caused by:
        0: Processing device `STM32G491xx`
        1: According to `FDCAN,FDCAN?`
        2: Processing peripheral `FDCAN2`
        3: According to `RXGFC`
        4: Could not find `FDCAN2:RXGFC. Present registers: .`
make: *** [svd/stm32g491.svd.patched] Error 1
```

Not sure what to do about this since the RXGFC register is definitely in the SVD.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -18,6 +18,7 @@
 * Fix ETH_MACFFR bitOffsets
 * GFXMMU LUT cluster
 * Fix writeConstraint bugs
+* STM32G491: Add FDCAN2 peripheral
 
 [#854]: https://github.com/stm32-rs/stm32-rs/pull/854
 

--- a/devices/stm32g491.yaml
+++ b/devices/stm32g491.yaml
@@ -1,5 +1,20 @@
 _svd: ../svd/stm32g491.svd
 
+_derive:
+  FDCAN2:
+    _from: FDCAN
+    baseAddress: 0x40006800
+
+FDCAN2:
+  add:
+    _interrupts:
+      FDCAN2_intr0:
+        description: FDCAN2_intr0
+        value: 86
+      FDCAN2_intr1:
+        description: FDCAN2_intr1
+        value: 87
+
 OPAMP:
   _copy:
     OPAMP6_CSR:

